tests/functional/api/test_groups.py

```diff
@@ -8,6 +8,21 @@
 @pytest.mark.functional
 class TestGroupAPI(object):
 
+    def test_create_group(self, app, user_with_token):
+        """Test a request to create a private group through the API"""
+
+        user, token = user_with_token
```

Comment:
This is fine, but an alternative would be for `token` and `user` to be two separate fixtures, with the `token` fixture using the `user` fixure, then tests could just take `user, token` as separate test method parameters.